### ISSUE TITLE
Only check inside the folder?

### ISSUE THREAD

**Issue by DankMemeGuy** [arsenetar/dupeguru#651]
I have a vast bulk of folders that I need to delete duplicates for. However, it should only check within the folder. So if I put in folders: 1,2,3,4 and Folders 1 and 3 have the same file, BUT within folders 1 and 3 there is only 1 file (no duplicates) then nothing should be removed. But if folder 4 has 2 copies of the same file then it would remove the duplicate.

File names are definitely not identical, but the size should be the same since they are identical images, not sure about MD5 hash.

DupeGuru seems to check all the folders when put in the program, instead of just the folder itself and its subfolders. Would be a nice feature to have.

**Comment by glubsy** [comment]
If I understand right, you are asking for a feature that would iterate each directory submitted by the user one by one, instead of all directories against each other?

**Comment by DankMemeGuy** [comment]
Yeah! Like if I submit 10 directories, the software would only check within each of the directory, not against all 10.

**Comment by FredWahl** [comment]
I can think of a neat UI solution to this request, the introduction of a ”Group” button. Simply select the paths and press that button. Repeat for the 10 directories you mention. Now, dupeguru should search for duplicates within each group. In this case, each path would be a group

There could be duplicates in different groups but they won’t be found. This should make dupeguru faster since there will be less files to consider.

**Comment by jeffkruczynski** [comment]
+1 to this request

**Comment by Suncatcher** [comment]
This is implemented as comparison method in [AllDup](https://www.alldup.de/alldup_help/alldup.php)

![image](https://user-images.githubusercontent.com/6388034/179392778-3f8e45bf-51d6-472a-ab9d-29be6313dd4f.png)

It allows to choose how to search for dups: within each source folder, between them and so on. It would be nice to have this implemented in dupeguru
